index.js

```diff
@@ -100,8 +101,29 @@ class Main {
     screen.key(['escape', 'q', 'C-c'], () => process.exit(0));
     // fixes https://github.com/yaronn/blessed-contrib/issues/10
     screen.key(['o', 'O'], () => {
-      const selectedLambdaFunctionName = this.table.rows.items[this.table.rows.selected].data[0];
-      return open(`https://${program.region}.console.aws.amazon.com/lambda/home?region=${program.region}#/functions/${selectedLambdaFunctionName}?tab=configuration`);
+      const selectedLambdaFunctionName = this.table.rows.items[
+        this.table.rows.selected
+      ].data[0];
+      return open(
+        `https://${program.region}.console.aws.amazon.com/lambda/home?region=${program.region}#/functions/${program.stackName}-${selectedLambdaFunctionName}?tab=configuration`,
+      );
+    });
+    screen.key(['d', 'D'], () => {
+      const selectedLambdaFunctionName = this.table.rows.items[
+        this.table.rows.selected
+      ].data[0];
+      if (provider === 'serverless') {
+        exec(
+          `serverless deploy -f ${selectedLambdaFunctionName} -r ${program.region} --aws-profile ${profile}`,
+          { cwd: location },
+          (error, stdout, stderr) => {
+            if (error) {
+              console.error(`stderr: ${stderr}`);
+              console.error(`exec error: ${error}`);
+            }
```

Comment:
shouldn't we log things in the log box? for example `stdout`

Reply:
I think that's best too